spring-cloud-gcp-autoconfigure/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -23,6 +23,12 @@
       "type": "java.lang.Boolean",
       "description": "Auto-configure Google Cloud Stackdriver tracing components.",
       "defaultValue": true
+    },
+    {
+      "name": "spring.cloud.gcp.security.iap.enabled",
+      "type": "java.lang.Boolean",
+      "description": "Auto-configure BLAH",
```

Comment:
Why do we need this file?

Reply:
IDE property autocompletion, I think?
But the default should not be to enable IAP, so I will remove the default value.